pom.xml

```diff
@@ -112,8 +112,8 @@
       <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
       <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
-      <maven.compiler.source>1.7</maven.compiler.source>
-      <maven.compiler.target>1.7</maven.compiler.target>
+      <maven.compiler.source>1.8</maven.compiler.source>
+      <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
The doc is outdated... we had complains from users, community members at the dev, and even peers at the company I work for. So, it was a big no from everybody!


We could require 1.8 at the server but at the client it's a bit more complicated.

I would wait a good reason to require 1.8. :) So far i don't have one.